### PR TITLE
feat(#951): Filter uncovered records

### LINKED ISSUE #951: [Labeling Rules] Filter uncovered records
It could be nice to explore/annotate records that are not covered by any defined rule. Some kind of filter that we can leave activated while navigating between modes

### PR DESCRIPTION
**Pull request by leiyre**
This PR allows filtering of records by uncovered by rules

Closes #951

### CODE REVIEW

File: frontend/components/commons/header/filters/FiltersList.vue [hunk 1]
```diff
@@ -60,6 +60,10 @@
               @apply="onApply"
             />
           </span>
+          <FilterUncoveredByRules
```
Comment (frascuchon) [inline frontend/components/commons/header/filters/FiltersList.vue]: Show this filter only if "Define Rules" modes is active, and dataset has rules already defined

File: frontend/components/commons/header/filters/FiltersList.vue [hunk 2]
```diff
@@ -59,6 +59,12 @@
               :filter="filter"
               @apply="onApply"
             />
+            <FilterUncoveredByRules
+              v-else-if="filter.type === 'checkbox' && showUncoveredByRulesFilter"
```
Comment (frascuchon) [inline frontend/components/commons/header/filters/FiltersList.vue]: Do we need the `filter.type === "checkbox"` condition?

**Review by frascuchon — CHANGES_REQUESTED** [review]
Inline comments + fix lint messages

File: frontend/components/commons/header/filters/FiltersList.vue (hunk 1 above)
Comment (leiyre) [reply frontend/components/commons/header/filters/FiltersList.vue]: @frascuchon but in "Explore" and "Annotation" mode, the filter must be also visible

Comment (frascuchon) [reply frontend/components/commons/header/filters/FiltersList.vue]: Yes, what I mean is enabled, sorry. You could see the uncovered records only if you can work with rules, so "Define Rules" mode is enabled

File: frontend/components/commons/header/filters/FiltersList.vue (hunk 2 above)
Comment (leiyre) [reply frontend/components/commons/header/filters/FiltersList.vue]: we can remove it

**Review by frascuchon — APPROVED** [review]

**Review by frascuchon — CHANGES_REQUESTED** [review]
fix build

**Review by frascuchon — APPROVED** [review]

**Pull request merged**